sdk/synapse/azure-analytics-synapse-accesscontrol/CHANGELOG.md

```diff
@@ -1,3 +1,3 @@
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2020-12-08)
 
-Initial release.
\ No newline at end of file
+Initial release.
```

Comment:
We generally include a standard template for first release:

## 1.0.0-beta.1 (20xx-xx-xx)

Version 1.0.0-beta.1 is a beta of our efforts in creating a client library that is developer-friendly, idiomatic to
the Java ecosystem, and as consistent across different languages and platforms as possible. The principles that guide
our efforts can be found in the
[Azure SDK Design Guidelines for Java](https://azure.github.io/azure-sdk/java_introduction.html).

### New Features

- Reactive streams support using [Project Reactor](https://projectreactor.io/).
- new features worth calling out

### Known Issues

- any known limitations of this release